src/main/java/com/bootcamp/demo/controller/CardController.java

```diff
@@ -1,24 +1,37 @@
 package com.bootcamp.demo.controller;
 
+import com.bootcamp.demo.model.Card;
 import com.bootcamp.demo.service.CardService;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.WriteResult;
+import com.google.firebase.cloud.FirestoreClient;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
-import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import java.util.List;
 
 @RestController
-@RequestMapping(value = "/card", produces = APPLICATION_JSON_VALUE )
+@RequestMapping("/card")
```

Comment:
`@RequestMapping(value = "/card", produces = APPLICATION_JSON_VALUE )` is the correct annotation for this REST Controller. Please change id back. Please be aware that there might have been errors generated when rebasing.

Reply:
Fixed